ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/control-has-associated-label': 'off',
```

Comment:
Requires elements to be labelled. While it can be as simple as adding `aria-label`, it's rarely the best solution, and the alternatives are rarely trivial. So again, this rule is important for new code but not so much for existing code.